internal/parser/references.go

```diff
@@ -4,28 +4,47 @@ import (
 	"strings"
 
 	"github.com/microsoft/typescript-go/internal/ast"
+	"github.com/microsoft/typescript-go/internal/checker"
 	"github.com/microsoft/typescript-go/internal/core"
 	"github.com/microsoft/typescript-go/internal/tspath"
 )
 
-func collectExternalModuleReferences(file *ast.SourceFile) {
+func (p *Parser) createSyntheticImport(text string, file *ast.SourceFile) *ast.Node {
+	externalHelpersModuleReference := p.factory.NewStringLiteral(text)
+	importDecl := p.factory.NewImportDeclaration(nil /*modifiers*/, nil /*importClause*/, externalHelpersModuleReference, nil)
 	// !!!
+	// addInternalEmitFlags(importDecl, InternalEmitFlags.NeverApplyImportHelper);
```

Comment:
this is probably important, but wasn't clear to me how to interact with `printer.EFNeverApplyImportHelper`